src/ReactResponsiveSelect.js

```diff
@@ -23,11 +24,11 @@ export default class ReactResponsiveSelect extends Component {
   handleKeyEvent = this.handleKeyEvent.bind(this);
 
   componentDidMount() {
-    const { options, selectedValue, selectedValues, name, multiselect, disabled } = this.props;
+    const { options, selectedValue, selectedValues, name, multiselect, disabled, altered } = this.props;
```

Comment:
I guess this is for consistency? It seems odd to me that the component would mount in the altered state

Reply:
Yes the component re-mounts. It does not seem odd to me. Do you think this should be looked at?

Adding `altered` prop is allowing for a container component to have control over displaying the altered state (adds a `.rrs--has-changed` class).

Reply:
Ah I guess this does actually make sense if you think of the container component controlling everything.

I think componentDidMount is only called the first time the component is rendered. The container would probably never need to initialize ReactResponsiveSelect in the altered state so I don't think we need to consume this prop here.

Reply:
I wondering whether it might be good to leave the `altered` prop in. It feels pretty harmless.

I'm thinking about one scenario around a half saved form. Whereas someone may wan't to use the `altered` state to signify an item that has been changed but not saved. Totally edge case, but I don't really know how people want to use the thing and can't see the harm in leaving it in.

Reply:
Fair point! It's definitely harmless and someone might use it.